scripts/run_tests.py

```diff
@@ -202,7 +203,9 @@ def startTest(self, test):
     if options.run_plugin_tests:
         test_identifiers.extend(get_plugin_tests())
 
-    IrodsController().restart(test_mode=True)
+    IrodsController().stop()
+    irods.lib.clear_shm()
```

Comment:
Have you seen a noticeable difference with this?

Reply:
Yes. I get far fewer failures with this change. Of course, I've left it in a separate commit so that it can be easily removed/not merged. I just found it useful and figured I'd put it out there.

Reply:
My gut reaction is great. Now I'm wondering if this would hide other issues.

It's not clear to me whether we should keep this in or not. Will think about it more.

Reply:
consider that we are hoping to get into k8s/autotesting soonish.   agree, still don't know if that's a vote for or against leaving this in.

Reply:
I'm pretty sure the problem will persist in k8s, so I don't think that should affect the decision.

The only issue I can think of that might be hidden is #6954 itself. If the issue occurs, it would manifest itself in (hopefully) only one test failing and this change would allow the next test to be executed to start fresh.

Reply:
i think i agree - that's the only known artifact at this time of what we **think** the problem is.

Reply:
I left a comment and moved `clear_shm` into `run_tests.py` since it's only meant to be used there (for now) anyway.

Reply:
Cool cool.